### PR TITLE
Login and createschedule

### PR DESCRIPTION
**Pull request by Rukeeo1**
Please i have finished the login and create schedule features....and tested them...
Screenshot 2019-08-07 at 1 33 41 PM

### CODE REVIEW

**Comment by Tyav** [comment]
login and createSchedule?

File: controllers/schedule.controller.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        to: time.to
+      },
+      slots: slots,
+      isClosed: isClosed,
+      mentorId: mentorId,
+      mentees: mentees
+    });
+
+  await schedule.save();
+    console.log('hy')
```
Comment (Tyav) [inline controllers/schedule.controller.js]: missed this?

**Comment by Rukeeo1** [comment]
Yes sir…. I accidentally worked on them in the same branch…that mistake won’t happen again… sorry boss.

File: controllers/schedule.controller.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    return res.json(sendResponse(200, 'Schedule Created', schedule, null));
+  } catch (error) {
+    console.log(error.message);
```
Comment (Tyav) [inline controllers/schedule.controller.js:41]: and this.

File: routes/auth.route.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const { login } = require('../controllers/user.controller');
+const router = express.Router(); // eslint-disable-line new-cap
+
+router.route('/');
+/** GET /api/v1/users - get all users */
+
+router.post('/login', (req, res) => {
```
Comment (Tyav) [inline routes/auth.route.js]: why is this here?

File: routes/schedule.route.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+const httpStatus = require('http-status');
+const userRoutes = require('./user.route');
+const authRoutes = require('./auth.route');
+const sendResponse = require('../helpers/response');
+const scheduleModel = require('../models/schedule.model');
+const modelCtrl = require('../controllers/schedule.controller');
+
+const router = express.Router();
+
+router.post('/', (req, res) => {
```
Comment (Tyav) [inline routes/schedule.route.js]: same as this, it should be a functionality in the `schedule.controller.js` file
```suggestion
router.post('/', modelCtrl.createSchedule)
```

File: controllers/schedule.controller.js (hunk 1 above)
Comment (Rukeeo1) [reply controllers/schedule.controller.js]: made all changes suggested.

**Review by Tyav — APPROVED** [review]
